src/components/Main/MainView/ChannelSideBar/ChannelSideBarHeaderName.vue

```diff
@@ -0,0 +1,42 @@
+<template>
+  <h2 :class="$style.container" :style="styles.container">
+    <span :class="$style.channelHash">#</span>
+    <span :class="$style.channel">{{ channelName }}</span>
+  </h2>
+</template>
+
+<script lang="ts">
+import { defineComponent, computed, reactive } from '@vue/composition-api'
+import { ChannelId } from '@/types/entity-ids'
+import { makeStyles } from '@/lib/styles'
+
+const useStyles = () =>
+  reactive({
+    container: makeStyles(theme => ({
+      color: theme.ui.primary
+    }))
+  })
+
+export default defineComponent({
+  name: 'ChannelSideBarHeaderName',
+  props: {
+    channelName: { type: String, required: true }
+  },
+  setup() {
+    const styles = useStyles()
+    return { styles }
+  }
+})
+</script>
+
+<style lang="scss" module>
+.container {
+  height: 100%;
+  font-size: 1.5rem;

```

Comment:
```suggestion
  font-size: 1.25rem;
```
見てみたらちょっと大きすぎたので:pray: